docs/static/azure-module.asciidoc

```diff
@@ -186,27 +157,67 @@ global configuration is effectively ignored.
 -----
 modules:
   - name: azure
-    var.elasticsearch.hosts: "localhost:9200"
-    var.kibana.host: "localhost:5601"
-    var.input.azure_event_hubs.threads: 8
-    var.input.azure_event_hubs.decorate_events: true
-    var.input.azure_event_hubs.consumer_group: logstash
+    var.elasticsearch.hosts: localhost:9200
+    var.kibana.host: localhost:5601
+    var.input.azure_event_hubs.decorate_events: true <1>
+    var.input.azure_event_hubs.threads: 9
+    var.input.azure_event_hubs.consumer_group: "logstash"
+    var.input.azure_event_hubs.storage_connection: "DefaultEndpointsProtocol=https;AccountName=instance1..."
     var.input.azure_event_hubs.event_hubs:
-      - ["name",                      "event_hub_connection",      "storage_connection",                                      "initial_position", "decorate_events"]
-      - ["insights-operational-logs", "Endpoint=sb://example1...", "DefaultEndpointsProtocol=https;AccountName=example1....", "HEAD",             "true"]
-      - ["insights-metrics-pt1m",     "Endpoint=sb://example2...", "DefaultEndpointsProtocol=https;AccountName=example2....", "TAIL",             "true"]
-      - ["insights-logs-errors",      "Endpoint=sb://example3...", "DefaultEndpointsProtocol=https;AccountName=example3....", "TAIL",             "false"]
-      - ["insights-operational-logs", "Endpoint=sb://example4...", "DefaultEndpointsProtocol=https;AccountName=example4....", "HEAD",             "true"]
+      - ["name",                                    "initial_position",  "storage_container",  "decorate_events",  "event_hub_connection"]                                   <2>
+      - ["insights-operational-logs",                 "TAIL",              "activity-logs1",    "true",             "DefaultEndpointsProtocol=https;AccountName=instance1"]
+      - ["insights-operational-logs",                 "HEAD",              "activity_logs2",<3>   "true",             "DefaultEndpointsProtocol=https;AccountName=instance2"] 
+      - ["insights-metrics-pt1m",                     "TAIL",              "dbmetrics",         "true",             "DefaultEndpointsProtocol=https;AccountName=instance1..."]
```

Comment:
the connect strings here should be Endpoint=:// (no reference to DefaultEndpointsProtocol..)

Reply:
We should be able to use the same connection strings used in the basic example.

Reply:
@jakelandis  Is this what you mean? 

<img width="934" alt="screen shot 2018-08-03 at 2 13 10 pm" src="https://user-images.githubusercontent.com/35154725/43658605-668dd9de-9727-11e8-95a8-83929ad413f1.png">

Reply:
the connection string should be the same between the two. ... the only that difference between the two rows should be the storage_container.